pysteps/nowcasts/steps.py

```diff
@@ -137,6 +137,11 @@ def forecast(R, V, n_timesteps, n_ens_members=24, n_cascade_levels=6,
       A string defining the FFT method to use (see utils.fft.get_method).
       Defaults to 'numpy' for compatibility reasons. If pyFFTW is installed,
       the recommended method is 'pyfftw'.
+    domain : {"spatial", "spectral"}
```

Comment:
Add reference to publication on spectral implementation of STEPS?

Reply:
Done.